code/game/objects/items/weapons/implant/implants/excelsior.dm

```diff
@@ -99,10 +99,13 @@
 	for(var/datum/antagonist/A in wearer.mind.antagonist)
 		if(A.id == antag_id)
 			A.remove_antagonist()
-
+	wearer.visible_message(SPAN_DANGER("As \the [src] is removed from \the [wearer]..."))
 	if(prob(66))
+		wearer.visible_message(SPAN_DANGER("\The [wearer]'s [part.name] violently explodes from within!"))
```

Comment:
What.


No.

Emojicracy this.

Reply:
This doesn't actually change functionality. The code works exactly the same as it does currently. The only difference is now there's flavor text.

Reply:
Hmm...alright. Fine.